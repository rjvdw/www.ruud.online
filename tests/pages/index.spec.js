'use strict'

const path = require('path')
const { loadPage } = require('../util')

test('home page matches snapshot', async () => {
  const homepage = await loadPage(
    path.join(__dirname, '../../_site/index.html'),
  )
  expect(homepage).toMatchSnapshot()
})
