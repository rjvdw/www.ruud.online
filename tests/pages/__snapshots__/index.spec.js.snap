// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page matches snapshot 1`] = `
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport"
          content="width=device-width,initial-scale=1"
    >
    <meta name="theme-color"
          content="#474069"
    >
    <meta name="color-scheme"
          content="light dark"
    >
    <title>
      ruud.online
    </title>
    <link rel="stylesheet"
          href="/css/main.css"
    >
    <link rel="stylesheet"
          href="/css/visual-enhancements.css"
    >
    <script>
      document.querySelectorAll('link[rel="preload"][as="style"]').forEach(el => el.onload = () => el.rel = 'stylesheet')
    </script>
    <meta name="author"
          content="Ruud van der Weide <rj@rdcl.dev>"
    >
    <meta name="description"
          content="Welcome to my homepage. My name is Ruud and I am a software developer from Nĳmegen, the Netherlands."
    >
    <meta name="google-site-verification"
          content="mGOj_7h6LAOAYpcB1MxuJFv6WopZ2_LhK-bJtI-klyM"
    >
    <meta name="msvalidate.01"
          content="D17D87D5B648EB4375349812B1FA3553"
    >
    <meta property="og:description"
          content="Welcome to my homepage. My name is Ruud and I am a software developer from Nĳmegen, the Netherlands."
    >
    <meta property="og:image"
          content="https://www.ruud.online/me.jpg"
    >
    <meta property="og:title"
          content="ruud.online"
    >
    <meta property="og:type"
          content="website"
    >
    <meta name="robots"
          content="index,follow"
    >
    <script type="application/ld+json">
      {"@context":"http://schema.org","@type":"Person","name":"Ruud van der Weide","email":["rj@rdcl.dev"],"url":"https://www.ruud.online","image":"https://www.ruud.online/me.jpg","sameAs":["https://mastodon.nl/@rdcl","https://github.com/rjvdw","https://keybase.io/rdcl","https://www.npmjs.com/~rdcl","https://nl.linkedin.com/in/rjvanderweide"]}
    </script>
  </head>
  <body class="home">
    <nav class="page-nav">
      <ul class="page-nav__container">
        <li class="page-nav__item page-nav__item--active">
          <span class="page-nav__item-link">
            Home
          </span>
        </li>
      </ul>
    </nav>
    <header class="page-header">
      <div class="page-header__container">
        <h1>
          Hello, World!
        </h1>
        <p>
          Welcome to my homepage. My name is Ruud and I am a software developer from Nĳmegen, the Netherlands.
        </p>
        <div class="page-header__profile-picture"
             role="presentation"
        >
        </div>
      </div>
    </header>
    <main class="page-main">
      <p>
        I have been developing software professionally since 2012. In my spare time I also like working on side projects, learning new programming languages, and trying out new web frameworks.
      </p>
      <p>
        Some (programming) languages I work with are: Java, Rust, Kotlin, Scala, Python, TypeScript, JavaScript/Node.js, HTML, CSS, C#, Ruby and Go.
      </p>
      <h2>
        Socials
      </h2>
      <ul>
        <li>
          <a href="https://mastodon.nl/@rdcl"
             rel="me noopener noreferrer"
             target="_blank"
          >
            Mastodon
          </a>
        </li>
        <li>
          <a href="https://github.com/rjvdw"
             rel="me noopener noreferrer"
             target="_blank"
          >
            GitHub
          </a>
        </li>
        <li>
          <a href="https://nl.linkedin.com/in/rjvanderweide"
             rel="me noopener noreferrer"
             target="_blank"
          >
            LinkedIn
          </a>
        </li>
      </ul>
      <h2>
        Certificates
      </h2>
      <ul>
        <li>
          <a href="https://www.credly.com/earner/earned/badge/8d179b16-64b2-4abe-8978-319ed8d40483"
             target="_blank"
             rel="noopener noreferrer"
          >
            Oracle Certified Associate, Java SE 8 Programmer
          </a>
        </li>
        <li>
          <a href="https://www.credly.com/earner/earned/badge/5087e8f9-d41d-453b-b16e-6fed220f3b0d"
             target="_blank"
             rel="noopener noreferrer"
          >
            Oracle Certified Professional, Java SE 8 Programmer
          </a>
        </li>
        <li>
          <a href="https://www.coursera.org/account/accomplishments/verify/7L9QPDZF4ESV"
             target="_blank"
             rel="noopener noreferrer"
          >
            Coursera – Functional Programming Principles in Scala
          </a>
        </li>
        <li>
          <a href="https://www.coursera.org/account/accomplishments/verify/DGTMN79P7VWJ"
             target="_blank"
             rel="noopener noreferrer"
          >
            Coursera – Functional Program Design in Scala
          </a>
        </li>
        <li>
          <a href="https://bcert.me/skkgftvcl"
             target="_blank"
             rel="noopener noreferrer"
          >
            Certified ScrumMaster
          </a>
        </li>
      </ul>
      <h2>
        Other Links
      </h2>
      <ul>
        <li>
          <a href="https://gist.github.com/rjvdw"
             target="_blank"
             rel="noopener noreferrer"
          >
            My gists on GitHub
          </a>
        </li>
        <li>
          <a href="/0AE37E45.asc">
            My public GPG key
          </a>
        </li>
        <li>
          <a href="https://todoist.com/r/ruud_van_der_weide_amadng"
             target="_blank"
             rel="noopener noreferrer"
          >
            My Todoist referral link
          </a>
        </li>
        <li>
          <a href="https://ref.fm/u10211109"
             target="_blank"
             rel="noopener noreferrer"
          >
            My Fastmail referral link
          </a>
        </li>
      </ul>
    </main>
  </body>
</html>
`;
